charts/dbaas-operator/values.yaml

```diff
@@ -84,7 +84,7 @@ image:
   repository: amazeeio/dbaas-operator
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: ""
+  tag: k8s_122
```

Comment:
do we have an updated image for 1.22 - the appVersion says v0.3.0? - we should remove this tag if so?